tremor-cli/src/test/assert.rs

```diff
@@ -209,6 +210,7 @@ pub(crate) fn process(
     Ok((s, elements))
 }
 
+#[allow(clippy::clippy::too_many_lines)]
```

Comment:
perhaps we can split out the different parts into their own functions?

Reply:
done